Android/app/src/go/doh/doh.go

```diff
@@ -125,23 +128,23 @@ func (t *transport) dial(network, addr string) (net.Conn, error) {
 	ips := t.ips.Get(domain)
 	confirmed := ips.Confirmed()
 	if confirmed != nil {
-		log.Debugf("Trying confirmed IP %s for addr %s", confirmed.String(), addr)
-		if conn, err = split.DialWithSplitRetry(t.dialer, tcpaddr(confirmed), nil); err == nil {
-			log.Infof("Confirmed IP %s worked", confirmed.String())
+		logging.Debug.Printf("Trying confirmed IP %s for addr %s\n", confirmed.String(), addr)
```

Comment:
These are all generating the strings, which we were not before. We've seen in the past that logging can consume a ton of memory and cpu.